eeg_cofitage/README

```diff
@@ -0,0 +1,71 @@
+﻿Overview
+--------
+
+This is the “COFITAGE” dataset. The aim is to collect sleep EEG data from 100 to 120 healthy older 
+participants (aged 50-70), in order to quantify sleep disruption, evaluate its impact on cognition
+and assess its relationship with brain amyloid and tau proteins deposition in preclinical AD.
+
+We would like to determine which sleep-wake regulation parameters are associated with biomarkers 
+and cognitive status at baseline and at a 24 months follow-up of cognitive performances.
+
+
+Citing this dataset
+-------------------
+
+This dataset and related analyses are unpublished. Please do not use/cite given data other than
+BIDS dataset example.
+
+
+Format
+------
+
+It is formatted according to BIDS version 1.2.0 and the BIDS standard extension
+for EEG (BEP006). The full description of the standard and extension which can be found here:
```

Comment:
BEP006 has been merged into BIDS, so there is no need to refer to it

`The dataset is formatted according to BIDS version 1.2.0.`

Reply:
also drop "and extension" from the next sentence.

Reply:
Will do

Reply:
Done